src/decoder/barcode_decoder.js

```diff
@@ -13,6 +13,7 @@ import UPCEReader from '../reader/upc_e_reader';
 import I2of5Reader from '../reader/i2of5_reader';
 import TwoOfFiveReader from '../reader/2of5_reader';
 import Code93Reader from '../reader/code_93_reader';
+import Code32Reader from '../reader/code_32_reader';
```

Comment:
I think if you add it to READERS here, at line 32, that might be all that you need to be able to reference it in Quagga.init()  and then we should be able to see some results?

Reply:
I'll try and pull this later today, i've been spending what little time i have to code on trying to get ean_reader to work in Typescript, but I think I can shift gears and give this a look

Reply:
OK thanks! Let me know if everything goes well!